Automattic/wp-calypso
# WPSC: Add Plugins tab

Needs Automattic/wp-super-cache#382.

![image](https://user-images.githubusercontent.com/96308/31049090-2e82cbe0-a62c-11e7-8b81-41d3cb0b0452.png)

To test:
* On your WPSC test site, check out the `add/plugins_endpoint` branch.
* On your local Calypso dev installation, check out this branch, and edit `client/extensions/wp-super-cache/app/constants.js` to change `minVersion` to `'1.5.5'` for the `PLUGINS` tab (otherwise the 'Plugins' tab will be hidden.)
* Test the extension:
  * Navigate to `http://calypso.localhost:3000/extensions/wp-super-cache/<yourJPsite>`
  * Verify that the 'Plugins' tab is present, and that toggling the switches enables and disables the respective plugins (just verify that they're reflected by the corresponding `/wp-admin/options-general.php?page=wpsupercache&tab=plugins` screen).
* Change the 'Plugins' tab's `minVersion` back to `'1.5.6'`, and verify that the 'Plugins' tab is now hidden. Also verify that directly landing on `/extensions/wp-super-cache/plugins/<yourJPsite>` redirects you to `/plugins/wp-super-cache/<yourJPsite>`.
